nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -114,15 +115,25 @@ class TensorAggregatorBase:
     aggregate them in terms of NNCFCollectorTensorProcessor operations.
     """
 
-    def __init__(self, tensor_processor: NNCFCollectorTensorProcessor, num_samples: Optional[int] = None):
+    def __init__(
+        self,
+        tensor_processor: NNCFCollectorTensorProcessor,
+        aggregation_axes: Optional[Tuple[int, ...]] = None,
+        num_samples: Optional[int] = None,
+    ):
         """
         :param tensor_processor: Backend-specific tensor processor.
+        :param aggregation_axes: Axes along which to operate.
+            Registered statistics are stacked along zero axis,
+            axes >=1 correspond to recieved statistic axes shifted left by 1.
         :param num_samples: Maximum number of samples to collect. Aggregator
             skips tensor registration if tensor registration was called num_samples times before.
             Aggregator never skips registration if num_samples is None.
         """
 
         self._tensor_processor = tensor_processor
+        self._aggregation_axes = (0,) if aggregation_axes is None else aggregation_axes
```

Comment:
Why not just keep `(0,)` as the default value since this argument is optional?

Reply:
I didn't find any preferences of default immutable attributes over None with following default attributes setup in PEP8 guideline.
I did it in a time when `aggregation_axes` was a mutable list. What way of default attributes do you prefer?